recipe_scrapers/goodhousekeeping.py

```diff
@@ -0,0 +1,93 @@
+# mypy: allow-untyped-defs
+
+import re
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class GoodHousekeeping(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "goodhousekeeping.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".ingredients-body h3",
+            ".ingredient-lists li",
+        )
```

Comment:
Hi Jpts, while this looks correct for the ingredient_grouping identifiers could you update one of your test cases to cover a page with this present?

The recipe for Vegetarian fajitas lists two separate sections for the recipe and "to serve" while this is exactly the intended use of the function it will definitely work and would be great to have a test case showing this part of the scraper is working as intended. A test file with the test coverage can be found [here](https://github.com/hhursev/recipe-scrapers/blob/9a10b98c61dec88d881012276a4c794c74387c4b/tests/test_bluejeanchef_2.py) if you need an example.

Reply:
Also if you could add an entry to the README.srt under the `Scrapers available for:` header in the correct spot